rskj-core/src/main/java/co/rsk/peg/FederationSupport.java

```diff
@@ -233,4 +236,19 @@ private boolean shouldFederationBeActive(Federation federation) {
         long federationAge = executionBlock.getNumber() - federation.getCreationBlockNumber();
         return federationAge >= bridgeConstants.getFederationActivationAge(activations);
     }
+
+    /**
+     * Get the Genesis Federation from a List of Genesis Federation Public Keys
+     *
+     * @return Federation
+     */
+    private Federation getGenesisFederation() {
+        final List<BtcECKey> genesisFederationPublicKeys = bridgeConstants.getGenesisFederationPublicKeys();
+        // IMPORTANT: BTC, RSK and MST keys are the same.
+        // Change upon implementation of the <INSERT FORK NAME HERE> fork.
```

Comment:
Why this comment?

Reply:
It's an old comment that I moved when I refactored the code. Should I remove it?

Reply:
Done!